docs/tutorials/build-a-dapp/prepare.md

```diff
@@ -26,7 +26,7 @@ Now you can proceed to set up the front-end template with these commands.
 
 ```bash
 # Clone the code from github
-git clone https://github.com/substrate-developer-hub/substrate-front-end-template
+git clone -b v2.0.0-alpha.3 https://github.com/substrate-developer-hub/substrate-front-end-template
```

Comment:
```suggestion
git clone -b v2.0.0-alpha.3 --depth 1 https://github.com/substrate-developer-hub/substrate-front-end-template
```

Reply:
It may be good to add depth. The reason is for this is that at some point in time, the repo contained a zip file that added a few extra MBs.

Reply:
Is it true for the front end template too? If so, I agree we should add depth here. I thought it wasn't a problem for front end template though.

Reply:
right, I didn't see that it was the front-end template. We don't need it there.